src/pal/prebuilt/inc/corprof.h

```diff
@@ -15151,8 +15151,6 @@ EXTERN_C const IID IID_ICorProfilerInfo9;
 /* [object][local][helpstring][version][uuid] */
 
 
-EXTERN_C const IID IID_ICLRProfiling;
```

Comment:
Anything under 'prebuilt' is generated by MIDL so the change will be wiped out the next time the file is regenerated.